server/datastore/mysql/migrations/tables/migration_test.go

```diff
@@ -0,0 +1,83 @@
+// Package tables holds fleet table migrations.
+//
+// Migrations can be tested with tests following the following format:
+//
+//	$ cat 20220208144831_AddSoftwareReleaseArchVendorColumns_test.go
+//
+//	[...]
+//	func TestUp_20220208144831(t *testing.T) {
+//		db := newDBConnForTests(t)
+//
+//		// Apply all migrations up to 20220208144831 (name of test), not included.
+//		applyUpToPrev(t, db)
+//
+//		// insert testing data, etc.
+//
+//		// The following will apply migration 20220208144831.
+//		applyNext(t, db)
+//
+//		// insert testing data, verify migration.
+//	}
+package tables
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+	"testing"
+
+	"github.com/jmoiron/sqlx"
+	"github.com/stretchr/testify/require"
+)
+
+// TODO(lucas): I'm copy pasting some of the mysql functionality methods here
+// otherwise we have import cycle errors.
+//
+// We need to decouple the server/datastore/mysql package,
+// it contains both the implementation of the fleet.Datastore and
+// MySQL functionality, and MySQL test functionality.
```

Comment:
:+1: , should create a ticket to address this tech debt at some point. Could be just the functionality specific to tests at first, e.g. in a `mysqltest` package.

Reply:
Ya, I started doing exactly that (same package name actually :) but it was getting a bit out of hand, probably better to do it on a separate PR :)

Reply:
:D yeah absolutely, separate PR makes sense to me.